cabal-install/Distribution/Client/Sandbox.hs

```diff
@@ -791,3 +797,18 @@ configCompilerAux' configFlags =
   configCompilerAuxEx configFlags
     --FIXME: make configCompilerAux use a sensible verbosity
     { configVerbosity = fmap lessVerbose (configVerbosity configFlags) }
+
+-- | Try to read the most recently configured compiler from the
```

Comment:
I wondered this too, which is why I factored it out as a separate function. But I haven't really investigated the other uses of `configCompilerAux'`, `configCompilerAuxEx`, and so on...